src/components/CertificateTemplates/tlds/sg/edu/ntu/common/certificate.js

```diff
@@ -0,0 +1,314 @@
+import { get } from "lodash";
+import { IMG_LOGO_NTU, IMG_CERTIFICATE_SEAL } from "./images";
```

Comment:
images are too large, please keep it under 500kb
culprit in particular is the red wax seal - it's unnecessarily high resolution

ideally use SVG vectors instead of raster formats for these

Reply:
I have updated the images. The file should be under 500kb.